ichnaea/service/locate.py

```diff
@@ -97,14 +107,14 @@ def map_data(data, client_addr=None):
     return mapped
 
 
-class AbstractResult(object):
-    """The result of a location provider query."""
+class AbstractLocation(object):
```

Comment:
I agree result wasn't the best of names here. I wrote Location first, but that leads to "PositionLocation", which really is the same concept twice and looked funny to me. I'm also not sure describing a country code as a "country location" matched that well. We might also get timezone as a third type, to we'd have a timezone location, which really is just a UTC offset or timezone code. Nitpicking, naming is hard, I don't have a better suggestion.